src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxInterceptorTest.java

```diff
@@ -1089,4 +1089,10 @@ public void callClosureElementOfMapAsMethod() throws Exception {
         assertEvaluate(new GenericWhitelist(), "a=hello,b=10", "def m = [ f: {a,b -> return \"a=${a},b=${b}\"} ]; m.f('hello',10)");
         assertEvaluate(new GenericWhitelist(), 2, "def m = [ f: {it.size()} ]; m.f(foo:0, bar:1)");
     }
+
+    @Issue("JENKINS-50906")
+    @Test
+    public void scriptBindingClosureVariableCall() throws Exception {
+        assertEvaluate(new GenericWhitelist(), true, "def func = { 1 }; this.func2 = { 1 }; return func() == func2();\n");
```

Comment:
A single variant is not enough to convince readers that the use of `(… "call", args)` works correctly as varargs. I would expect to see closures with zero, one, and two or more parameters, as well as implicit `it`.

Reply:
Fair enough - will do.